src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,73 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useCallback, useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+export type Props = {
+  onAdd(movie: Movie): void,
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [movieTitle, setMovieTitle] = useState('');
+  const [movie, setMovie] = useState<Movie | null>();
+  const [loader, setLoader] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
it's better to name boolean variables starting with `is`, `has` etc. https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig